metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -308,6 +318,7 @@ func (sink *StackdriverSink) sendRequest(req *sd_api.CreateTimeSeriesRequest) {
 	timeseriesSent.
 		WithLabelValues(strconv.Itoa(responseCode)).
 		Add(float64(len(req.TimeSeries)))
+	requestsLatency.Observe(float64(time.Since(startTime)) / float64(time.Millisecond))
```

Comment:
I strongly feed about `float64(time.Since(startTime)) -> time.Since(startTime).Seconds()`

But not so sure about `float64(time.Millisecond) -> time.Millisecond.Seconds()`, but it's still better IMO